### PR TITLE
Update commands for MS MARCO V2 reproduction

### PR DESCRIPTION
**Pull request by lintool**

### CODE REVIEW

**Comment by lintool** [comment]
hey @MXueguang this is ready for review. Tweaks for MS MARCO v2 documentation - uniCOIL and TILDE. Fixed inconsistent paths and other minor issues.

**Review by MXueguang — APPROVED** [review]

File: docs/experiments-msmarco-v2-unicoil-tilde-expansion.md [hunk 1]
```diff
@@ -26,31 +26,35 @@ First, we need to download and extract the MS MARCO V2 passage dataset with uniC
 
 ```bash
 # Alternate mirrors of the same data, pick one:
-wget https://rgw.cs.uwaterloo.ca/JIMMYLIN-bucket0/data/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar -P collections/
-wget https://vault.cs.uwaterloo.ca/s/tb3m3J45HFJNAbq/download -O collections/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar
+wget https://rgw.cs.uwaterloo.ca/JIMMYLIN-bucket0/data/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar -P collections/
+wget https://vault.cs.uwaterloo.ca/s/tb3m3J45HFJNAbq/download -O collections/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar
 
-tar -xvf collections/msmarco-v2-passage-unicoil-tilde-expansion-b8.tar -C collections/
+tar -xvf collections/msmarco-passage-v2-unicoil-tilde-expansion-b8.tar -C collections/
```
Comment (MXueguang) [inline docs/experiments-msmarco-v2-unicoil-tilde-expansion.md:32]: just to confirm, we are doing `msmarco-passage-v2` here for this file only, right?

**Review by MXueguang — APPROVED** [review]

File: docs/experiments-msmarco-v2-unicoil-tilde-expansion.md (hunk 1 above)
Comment (lintool) [reply docs/experiments-msmarco-v2-unicoil-tilde-expansion.md:32]: Yes, this file was created before we standardized our convention. The underlying `tar` file decompresses to `msmarco-passage-v2`. But at least uniCOIL noexp and uniCOIL + TILDE are consistent.

I think It's too much work to go back and change everything... that would requiring renaming the underlying data files, re-compressing, re-uploading, etc.

Comment (MXueguang) [reply docs/experiments-msmarco-v2-unicoil-tilde-expansion.md:32]: sg

**Pull request merged**